llpc/test/shaderdb/gfx9/ExtShaderInt8_TestShiftOp_lit.comp

```diff
@@ -23,9 +23,9 @@ void main()
 ; RUN: amdllpc -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
 ; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
 ; SHADERTEST-LABEL: {{^// LLPC}} SPIR-V lowering results
-; SHADERTEST: %{{[0-9]*}} = ashr i8 %{{[0-9]*}}, 3
-; SHADERTEST: %{{[0-9]*}} = lshr i8 %{{[0-9]*}}, 3
-; SHADERTEST: %{{[0-9]*}} = shl i8 %{{[0-9]*}}, 3
+; SHADERTEST: %{{[0-9]*}} = ashr i8 %.fr{{[0-9]*}}, 3
```

Comment:
Since it is not immediately obvious what the `.fr` operand is, could you include the freeze instructions in the test please?

Reply:
Done. This shows an interesting effect. LLPC inserts the `freeze` after the shift, but InstCombine moves it before the shift if it can see that the shift amount operand is in-range.